plugins/jobs/server/models/job.py

```diff
@@ -28,19 +28,23 @@
 
 
 class Job(AccessControlledModel):
+
     def initialize(self):
         self.name = 'job'
         compoundSearchIndex = (
             ('userId', SortDir.ASCENDING),
-            ('created', SortDir.DESCENDING)
+            ('created', SortDir.DESCENDING),
+            ('type', SortDir.ASCENDING),
+            ('status', SortDir.ASCENDING)
         )
         self.ensureIndices([(compoundSearchIndex, {}), 'created'])
 
         self.exposeFields(level=AccessType.READ, fields={
             'title', 'type', 'created', 'interval', 'when', 'status',
             'progress', 'log', 'meta', '_id', 'public', 'async', 'updated', 'timestamps'})
 
-        self.exposeFields(level=AccessType.SITE_ADMIN, fields={'args', 'kwargs'})
+        self.exposeFields(level=AccessType.SITE_ADMIN,
```

Comment:
Why this change?  Is your editor somehow set to only allow 80 chars or to reformat lines with more than 80 chars?  We allow 100 chars in Girder Python.

It isn't a big deal, but I wanted to point it out because whitespace changes can make the diff take longer to read through and can be a distraction while reviewing.